t2t_sb/lightning_module.py

```diff
@@ -106,7 +106,10 @@ def configure_optimizers(self):
         )
         monitor = (
             "val_loss"
-            if self.trainer.fit_loop.epoch_loop.val_loop._data_source.is_defined()
+            if (
+                hasattr(self.trainer.fit_loop.epoch_loop.val_loop, "_data_source") and 
+                self.trainer.fit_loop.epoch_loop.val_loop._data_source.is_defined()
+            )
```

Comment:
I get an error with this not being an attribute when running. Maybe a lightning version mismatch or something

Reply:
yes you prob. have an older version of lightning...